keyboards/doio/kb38/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+# forgot what goes here
```

Comment:
This file should be removed.

Reply:
Still outstanding.

Reply:
> Still outstanding.

Sorry about that, forgot about that file. It's removed now.